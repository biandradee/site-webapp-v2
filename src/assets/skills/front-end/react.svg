<svg width="80" height="80" viewBox="0 0 80 80" fill="none" xmlns="http://www.w3.org/2000/svg">
<path d="M60.3601 39.9997C60.3601 44.8201 51.243 48.727 39.9964 48.727C28.7499 48.727 19.6328 44.8201 19.6328 39.9997C19.6328 35.1794 28.7499 31.2725 39.9964 31.2725C51.243 31.2725 60.3601 35.1794 60.3601 39.9997Z" stroke="white" stroke-width="2"/>
<path d="M30.302 57.9052C34.5405 60.2034 42.3165 54.0448 47.6751 44.1568C53.0307 34.2659 53.9383 24.3896 49.6998 22.0943C45.4613 19.7961 37.6853 25.9546 32.3267 35.8426C26.9711 45.7336 26.0634 55.6099 30.302 57.9052Z" stroke="white" stroke-width="2"/>
<path d="M49.6998 57.9052C45.4613 60.2034 37.6853 54.0448 32.3267 44.1568C26.9711 34.2659 26.0634 24.3896 30.302 22.0943C34.5405 19.7961 42.3165 25.9546 47.6751 35.8426C53.0307 45.7336 53.9383 55.6099 49.6998 57.9052Z" stroke="white" stroke-width="2"/>
<path d="M37.0859 39.9999C37.0859 40.7715 37.3924 41.5114 37.938 42.057C38.4836 42.6025 39.2235 42.909 39.995 42.909C40.7666 42.909 41.5065 42.6025 42.0521 42.057C42.5976 41.5114 42.9041 40.7715 42.9041 39.9999C42.9041 39.2284 42.5976 38.4884 42.0521 37.9429C41.5065 37.3973 40.7666 37.0908 39.995 37.0908C39.2235 37.0908 38.4836 37.3973 37.938 37.9429C37.3924 38.4884 37.0859 39.2284 37.0859 39.9999Z" stroke="white" stroke-width="2"/>
</svg>
