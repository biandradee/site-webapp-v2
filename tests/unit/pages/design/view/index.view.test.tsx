import React from "react";
import { render } from "@testing-library/react";
import DesignPage from "../../../../../src/pages/design/view/index.view";

describe("Design Page View", () => {
  it("should render header correctly", () => {
    const links = [
      { label: "aboutUs", path: "/about-us", onClick: jest.fn() },
      { label: "initiatives", path: "/initiatives", onClick: jest.fn() },
    ];

    const { getByAltText } = render(<DesignPage links={links} />);

    expect(getByAltText("Sou Junior Logo")).toBeInTheDocument();
  });

  it("should match snapshot", () => {
    const links = [
      { label: "aboutUs", onClick: jest.fn() },
      { label: "initiatives", onClick: jest.fn() },
    ];

    const { container } = render(<DesignPage links={links} />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
