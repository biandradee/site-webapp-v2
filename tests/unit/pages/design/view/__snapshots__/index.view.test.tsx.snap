// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`Design Page View should match snapshot 1`] = `
<header
  class="sc-bRKDuR gJwjwH"
>
  <img
    alt="Sou Junior Logo"
    class="sc-hvigdm jCrChq"
    src="test-file-stub"
  />
  <nav
    class="sc-fhHczv gdjGrq"
  >
    <a
      class="sc-ggWZvA hthFqS"
    >
      Sobre n√≥s
    </a>
    <a
      class="sc-ggWZvA hthFqS"
    >
      Nossas iniciativas
    </a>
  </nav>
</header>
`;
